maps/atlas.dmm

```diff
@@ -20178,6 +20179,27 @@
 	},
 /turf/simulated/wall/auto/reinforced/supernorn,
 /area/station/science/storage)
+"YB" = (
+/obj/table/auto,
+/obj/machinery/camera{
+	c_tag = "autotag";
+	dir = 6;
+	name = "autoname - SS13";
+	pixel_y = 20;
+	tag = ""
+	},
+/obj/item/luggable_computer/personal,
+/obj/machinery/cell_charger,
```

Comment:
fairly sure you fucked up an upstream pull aat some point

Reply:
This being outdated doesn't override my concern

Reply:
its all gone now its a prefab!